.npmrc

```diff
@@ -0,0 +1,2 @@
+@fortawesome:registry=https://npm.fontawesome.com/
+//npm.fontawesome.com/:_authToken=2236DA7E-56B2-4138-9ED2-F81FECC44495
```

Comment:
I've made the .npmrc file ignored in Git. Secrets shouldn't be comitted as this a public repository. I have regenerated this token.